packages/twenty-front/src/modules/object-record/record-board/record-board-card/components/RecordBoardCard.tsx

```diff
@@ -293,7 +288,7 @@ export const RecordBoardCard = ({
                     accent="tertiary"
```

Comment:
icon should be conditionally rendered
eg - 
   Icon={isCardInCompactMode ? IconEye : IconEyeOff}